benchmark/utils/preconditioners.hpp

```diff
@@ -55,6 +59,18 @@ DEFINE_double(sor_relaxation_factor, 1.0,
 DEFINE_bool(sor_symmetric, false,
             "Apply the SOR preconditioner symmetrically, i.e. use SSOR");
 
+DEFINE_bool(pgm_deterministic, false,
+            "Use deterministic computation of the aggregated group within PGM");
+
+DEFINE_uint32(
+    mg_max_num_levels, false,
+    "The maximum number of levels to use for the Multigrid preconditioner");
+
+DEFINE_double(mg_tolerance, false, "The tolerance for the coarse solver");
+
+DEFINE_uint32(mg_max_iters, false,
+              "The max number of iterations for the coarse solver");
```

Comment:
To me, this looks like we reached the limit of what we can do through the CLI. I think for now this is fine, but we should consider in the long term to also allow our json config to define these parameters.